### PR TITLE
Update input component

### PR DESCRIPTION
**Pull request by rolfheij-sil**
This change is [Reviewable](https://reviewable.io/reviews/paranext/paranext-core/1173)

### CODE REVIEW

File: lib/platform-bible-react/src/components/shadcn-ui/input.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ export const Input = React.forwardRef<HTMLInputElement, InputProps>(
       <input
         type={type}
         className={cn(
+          // Removed pr-h-10, as it might the component look very different than other shadcn components. This was not discussed this with UX
```
Comment (tombogle) [inline lib/platform-bible-react/src/components/shadcn-ui/input.tsx:12]: I think you meant to  say: Removed pr-h-10, as it made the component look very different than other shadcn components. This was not discussed with UX.

**Review by tjcouch-sil — CHANGES_REQUESTED** [review]
:lgtm:

> Reviewed 5 of 5 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-core/1173#-:-O7o85t01FERFsS2gGN1:bnfp4nl)* status: all files reviewed, 2 unresolved discussions (waiting on @rolfheij-sil)

___
*[`lib/platform-bible-react/src/components/shadcn-ui/input.tsx` line 12 at r1](https://reviewable.io/reviews/paranext/paranext-core/1173#-O7o9Nwa5fW6WdjiKfH3:-O7o9Nwa5fW6WdjiKfH4:b-38nwdj) ([raw file](https://github.com/paranext/paranext-core/blob/4d93d5f746e904680c0983fcbacb94175b3c0321/lib/platform-bible-react/src/components/shadcn-ui/input.tsx#L12)):*
I don't understand from this comment what is the reason for removing this. Also it would be good to indicate what the change does to the component. Sorry I didn't mention this before. Did you say you removed it so we could change the height in other places? If so, maybe this could be an effective comment:

> Removed pr-h-10, making the input a few pixels shorter, because we need to be able to resize this easily in other places.

Why not discuss with UX? Could be as simple as passing by Sebastian (hopefully) real quick. Here's a comparison image for the sake of discussing with UX. I tested with smaller font size and determined that smaller font size does not change the height. Also, you can't type multiple lines in this, so that wouldn't have an effect on the height either.

**Review by tjcouch-sil — COMMENTED** [review]
Sorry, accidental LGTM :grimacing: I did request some changes

> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-core/1173#-:-O7oBmav3sRDAVwqnvFT:b-hiftbl)* status: all files reviewed, 2 unresolved discussions (waiting on @rolfheij-sil)

**Review by tjcouch-sil — DISMISSED** [review]
:lgtm:

> Reviewed 5 of 5 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-core/1173#-:-O7oop928n7FI4KSw4Yl:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @rolfheij-sil)

**Review by rolfheij-sil — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-core/1173)* status: all files reviewed, 1 unresolved discussion (waiting on @tombogle)

___
*[`lib/platform-bible-react/src/components/shadcn-ui/input.tsx` line 12 at r1](https://reviewable.io/reviews/paranext/paranext-core/1173#-O7o9Nwa5fW6WdjiKfH3:-O7oIgUO4z1ZK-L735TF:b-bufvvp) ([raw file](https://github.com/paranext/paranext-core/blob/4d93d5f746e904680c0983fcbacb94175b3c0321/lib/platform-bible-react/src/components/shadcn-ui/input.tsx#L12)):*
<details><summary><i>Previously, tjcouch-sil (TJ Couch) wrote…</i></summary><blockquote>

I don't understand from this comment what is the reason for removing this. Also it would be good to indicate what the change does to the component. Sorry I didn't mention this before. Did you say you removed it so we could change the height in other places? If so, maybe this could be an effective comment:

> Removed pr-h-10, making the input a few pixels shorter, because we need to be able to resize this easily in other places.

Why not discuss with UX? Could be as simple as passing by Sebastian (hopefully) real quick. Here's a comparison image for the sake of discussing with UX. I tested with smaller font size and determined that smaller font size does not change the height. Also, you can't type multiple lines in this, so that wouldn't have an effect on the height either.


</blockquote></details>

This is the behavior I observed

with pr-h-10



without pr-h-10



However, I just ran it by Sebastian, and it seems like this was not related to just pr-h-10, I think it had more to do with pr-twp. So I will actually restore pr-h-10, as that seems to be more in line with the other shadcn components
___
*[`lib/platform-bible-react/src/components/shadcn-ui/input.tsx` line 12 at r1](https://reviewable.io/reviews/paranext/paranext-core/1173#gh-1778837427:-O7oCkbK4s_kj5Qnyv_e:b-szwlbs) ([raw file](https://github.com/paranext/paranext-core/blob/4d93d5f746e904680c0983fcbacb94175b3c0321/lib/platform-bible-react/src/components/shadcn-ui/input.tsx#L12)):*
<details><summary><i>Previously, tombogle (Tom Bogle) wrote…</i></summary><blockquote>

I think you meant to  say: Removed pr-h-10, as it made the component look very different than other shadcn components. This was not discussed with UX.
</blockquote></details>

Thanks for checking my spelling. I can't do words today. The comment has been removed, so that should fix it too

**Comment by tombogle** [comment]
*[`lib/platform-bible-react/src/components/shadcn-ui/input.tsx` line 12 at r1](https://reviewable.io/reviews/paranext/paranext-core/1173#gh-1778837427:-O7p_2uM9LM--j1tVN5K:b-3r0f3m) ([raw file](https://github.com/paranext/paranext-core/blob/4d93d5f746e904680c0983fcbacb94175b3c0321/lib/platform-bible-react/src/components/shadcn-ui/input.tsx#L12)):*
<details><summary><i>Previously, rolfheij-sil (Rolf Heij) wrote…</i></summary><blockquote>

Thanks for checking my spelling. I can't do words today. The comment has been removed, so that should fix it too
</blockquote></details>

My understanding from the component guidelines document was that we were supposed to document this kind of change. No?

**Review by rolfheij-sil — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-core/1173)* status: all files reviewed, 1 unresolved discussion (waiting on @tombogle)

___
*[`lib/platform-bible-react/src/components/shadcn-ui/input.tsx` line 12 at r1](https://reviewable.io/reviews/paranext/paranext-core/1173#gh-1778837427:-O7pwTD40wL8F08aMy0w:bnox271) ([raw file](https://github.com/paranext/paranext-core/blob/4d93d5f746e904680c0983fcbacb94175b3c0321/lib/platform-bible-react/src/components/shadcn-ui/input.tsx#L12)):*
<details><summary><i>Previously, tombogle (Tom Bogle) wrote…</i></summary><blockquote>

My understanding from the component guidelines document was that we were supposed to document this kind of change. No?
</blockquote></details>

Yes, we should document any changes we make to shadcn code. However, this issue was about adding a comment to a change that had been made in a previous pr. Today I found out that change was actually unneccessary, and so I reverted the change by putting pr-h-10 back. So the code is back to how we got it from shadcn when we installed the component, and so a comment is not needed here

**Review by tjcouch-sil — DISMISSED** [review]
> Reviewed 4 of 4 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-core/1173)* status: all files reviewed, 1 unresolved discussion (waiting on @tombogle)

**Review by tjcouch-sil — APPROVED** [review]
> Reviewed 4 of 4 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-core/1173)* status: all files reviewed, 1 unresolved discussion (waiting on @tombogle)

**Review by tombogle — APPROVED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/paranext/paranext-core/1173)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved

**Pull request merged**